azkaban-common/src/main/java/azkaban/jobtype/JobTypeManager.java

```diff
@@ -55,7 +55,7 @@
   private final ClusterRouter clusterRouter;
   private JobTypePluginSet pluginSet;
   // Only used to load keyStore.
-  private Props keyStoreLoadProps = null;
+  private Props commonPluginLoadPropsCache = null;
```

Comment:
Can you please rename it to cachedCommonPluginLoadProps? Sorry I did not come up with this name earlier. I just thought about this.